EP2/main.cpp

```diff
@@ -0,0 +1,93 @@
+#include <iostream>
+#include <string>
+
+#include "Serie.h"
+#include "SerieTemporal.h"
+#include "Ponto.h"
+#include "Eixo.h"
+#include "EixoDinamico.h"
+#include "InterfaceSerial.h"
+#include "Grafico.h"
+
+using namespace std;
+
+#define COMM "\\\\.\\COM3"
+
+int main() {
+    InterfaceSerial* is = new InterfaceSerial(COMM);
+    is->inicializar();
+    string nomeDaSerie, canalX, canalY;
+    int numX, numY;
+    char tipoX, tipoY;
+
+    // Obtem o nome da Serie e os canais escolhidos
+    cout << "Informe o nome da serie: ";
+    cin >> nomeDaSerie;
+
+    cout << "Escolha o canal X: " << endl;
+    for(int i = 1; i <= is->getQuantidadeDeCanais(); i++)
+        cout << i << ") " << *(is->getNomeDosCanais() + (i - 1)) << endl;
+    cin >> numX;
+    canalX = *(is->getNomeDosCanais() + (numX - 1));
+
+    cout << "Escolha o canal Y:" << endl;
+    for(int i = 1; i <= is->getQuantidadeDeCanais(); i++)
+        cout << i << ") " << *(is->getNomeDosCanais() + (i - 1)) << endl;
+    cin >> numY;
+    canalY = *(is->getNomeDosCanais() + (numY - 1));
+
+    // Obtem o numero de Pontos a adicionar
+    int quantidade;
+    cout << "Obter quantos pontos? ";
+    cin >> quantidade;
+    Serie* s = new Serie(nomeDaSerie, canalX, canalY);
+
```

Comment:
Falta a impressão de _"Obtendo os pontos"_, ao longo da obtenção dos pontos da **Serie**. 


`cout << "Obtendo os pontos" << endl;` 
`    for (int i = 0; i < quantidade; i++) {`
____ `        is->atualizar();`
____ `        serie->adicionar(is->getValor(canalX), is->getValor(canalY));`
`    }`